rails-time-track/app/serializers/project_serializer.rb

```diff
@@ -1,7 +1,13 @@
 class ProjectSerializer < ActiveModel::Serializer
-  attributes :id, :name, :client, :category, :start_date, :end_date, :closed, :estimated_cost, :real_cost, :members
+  attributes :id, :name, :client, :category, :start_date, :end_date, :closed, :estimated_cost, :real_cost, :members, :weekly
 
   def members
     self.object.users.select("users.id, users.name, users.role")
   end
+
+
+  def weekly
+    self.object.weekly_project_reports.select("weekly_project_reports.id, weekly_project_reports.week, weekly_project_reports.estimated_cost, weekly_project_reports.real_cost")
```

Comment:
Have you thought refactor this piece of code in order to make it more easy to read?

Reply:
Wondering how I could do such...
That line is like:
```
SELECT  weekly_project_reports.id, weekly_project_reports.week, weekly_project_reports.estimated_cost, weekly_project_reports.real_cost FROM "weekly_project_reports" WHERE "weekly_project_reports"."project_id" = "project_id"
```
it add an array with the necessary `weekly_project_reports` for the projects. Looks bulky but is quite direct.